internal/controller/gitrepository_controller_test.go

```diff
@@ -1543,15 +1563,122 @@ func TestGitRepositoryReconciler_verifyCommitSignature(t *testing.T) {
 			beforeFunc: func(obj *sourcev1.GitRepository) {
 				obj.Spec.Interval = metav1.Duration{Duration: interval}
 				obj.Spec.Verification = &sourcev1.GitRepositoryVerification{
-					Mode: "head",
+					Mode: "HEAD",
```

Comment:
its using the constants: https://github.com/fluxcd/source-controller/pull/1187/files#diff-d3d1cb112880781b03e6c0af10a1025598f7c90c95f88a0f931d03d4e34b3c9dR1752 (except for the "head" test case, since we don't have a constant for that and its a outlier)

Reply:
oh are you referring to `Test_requiresVerification`?

Reply:
yes, gotcha